crates/whiskers/src/grid/cell.rs

```diff
@@ -0,0 +1,27 @@
+//!
+use vsvg::Point;
+
+/// Grid cell data structure
+///
+/// It holds grid coordinate values, canvas position, size info
+/// and optional data one could pass to it when using grid system
+/// to implement one's own algorithm (like tiles for example).
+pub struct GridCell<T> {
+    /// Column index
+    pub column: usize,
+    /// Row index
+    pub row: usize,
+    /// Canvas position
+    pub canvas_position: Point,
+    /// User-defined data
+    pub data: Option<T>,
```

Comment:
The implementation will be much cleaner without having to deal with any user data

Reply:
good idea with the builder pattern. Idk why, but I used it when implementing the very same grid in JS, but forgot it when doing same in Rust.